openapi/kas-fleet-manager.yaml

```
@@ -951,6 +985,77 @@ components:
           type: boolean
           nullable: true
+    SupportedKafkaInstanceTypesList:
+      allOf:
+        - $ref: "#/components/schemas/List"
+        - type: object
+          example:
+            kind: "SupportedKafkaInstanceTypesList"
+            page: "1"
+            size: "1"
+            total: "1"
+            item:
+              $ref: "#/components/examples/SupportedKafkaInstanceTypeExample"
+          properties:
+            items:
+              type: array
+              items:
+                allOf:
+                  - $ref: "#/components/schemas/SupportedKafkaInstanceType"
+    SupportedKafkaInstanceType:
+      description: "Supported Kafka instance type"
+      properties:
+        kind:
+          description: "Indicates the type of this object. Will be 'SupportedKafkaInstanceType' link."
+          type: string
+        id:
+          description: "Unique identifier of the Kafka instance type."
+          type: string
+        sizes:
+          description: " A list of Kafka instance sizes available for this instance type"
+          type: array
+          items:
+            allOf:
+              - $ref: "#/components/schemas/SupportedKafkaSize"
+    SupportedKafkaSize:
+      description: "Supported Kafka Size"
+      type: object
+      properties:
+        kind:
+          description: "Indicates the type of this object. Will be 'SupportedKafkaSize' link."
+          type: string
+        id:
+          description: "Unique identifier of this Kafka instance size."
+          type: string
+        ingressThroughputPerSec:
+          description: "Ingress throughput per second available to this Kafka instance size."
+          type: string
+        egressThroughputPerSec:
+          description: "Egress throughput per second available to this Kafka instance size."
+          type: string
+        totalMaxConnections:
+          description: "Total amount of connections available to this Kafka instance size."
+          type: int
+        maxDataRetentionSize:
+          description: "Maximum data storage available to this Kafka instance size."
+          type: string
+        maxPartitions:
+          description: "Total amount of partitions available to this Kafka instance size."
+          type: int
+        maxDataRetentionPeriod:
+          description: "Maximum data retention period available to this Kafka instance size."
+          type: string
+        maxConnectionAttemptsPerSec:
+          description: "Maximium connection attempts per second available to this Kafka instance size."
+          type: int
+        quotaConsumed:
+          description: "Quota consumed by this Kafka instance size."
+          type: int
+        quotaType:
+          description: "Quota type used by this Kafka instance size."
+          type: string
+        capacityConsumed:
+          description: "Data plane cluster capacity consumed by this Kafka instance size"
```

```suggestion
        capacityConsumed:
          description: "Data plane cluster capacity consumed by this Kafka instance size"
          type: int
```
adding a missing `type` field.

---

Thanks @machi1990